JSDemos/Demos/Tooltip/Overview/description.md

```diff
@@ -1 +1,16 @@
-The Tooltip component displays a&nbsp;tooltip for a&nbsp;specified element on&nbsp;the page. This demo shows how you can customize the tooltip and animate its appearance on&nbsp;the page.
\ No newline at end of file
+The Tooltip component displays a tooltip near an element on the page. To bind the Tooltip to the element, use the [target](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#target) property. 
+
+## Show and Hide the Tooltip
+
+Use two-way binding on the [visible](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#visible) property of the Tooltip and assign a variable to this property. In the target element markup, assign a function that changes this variable to the specific [DOM events](https://developer.mozilla.org/en-US/docs/Web/Events#event_listing) that trigger the visibility of a tooltip.
+
+To hide the Tooltip when a user clicks outside it, use the [hideOnOutsideClick](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#hideOnOutsideClick) property.
+
+## Customize and Animate the Tooltip
+
+Assign Tooltip's content in the HTML markup. You can also use [contentTemplate](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#contentTemplate) to customize the Tooltip's content.
```

Comment:
Please do not use API members as ordinary sentence members. And don't forget about articles.

```suggestion
Assign the Tooltip's content in the HTML markup. Alternatively, you can use the [content template](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#contentTemplate) to customize the Tooltip's content.
```